app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) != 3:
+        return
+
+    cp, source, target = parts
+
+    if source == target:
+        return
+
+    with open(source, "r") as file_in, open(target, "w") as file_out:
+        for line in file_in:
+            file_out.write(line)
```

Comment:
You don't need loop here
```suggestion
        file_out.write(file_in.read())
```